src/ZooKeeper/ZooKeeperCommon.h

```diff
@@ -474,6 +479,8 @@ struct ZooKeeperListResponse : ListResponse, ZooKeeperResponse
 
 struct ZooKeeperSimpleListResponse final : ZooKeeperListResponse
```

Comment:
The processing of `ZooKeeperSimpleListRequest` and `ZooKeeperListRequest`  are basically same, so do  `ZooKeeperSimpleListResponse` and `ZooKeeperListResponse`. I think It's better 
for `ZooKeeperListResponse` inheritance from `ZooKeeperListResponse`, so that we can process `SimpleList` and `List` easily.